src/main/java/com/google/sps/auth/Auth.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.meltingpot.auth;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseAuthException;
+import com.google.firebase.auth.FirebaseToken;
+import com.google.sps.meltingpot.data.DBUtils;
+
+public class Auth {
+  private static FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
+
+  public static FirebaseToken verifyIdToken(String idToken) {
+    try {
+      FirebaseToken decodedToken = firebaseAuth.verifyIdToken(idToken, true);
+      return decodedToken;
+    } catch (IllegalArgumentException e) {
```

Comment:
I think you should be able do to catch (IllegalArgumentException|FirebaseAuthException e) { ... } to not duplicate this block

Reply:
I didn't know that was a feature. Interesting.

Done in 13fde54.